### ISSUE TITLE
port twisted.python.systemd to python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7566]
|<img alt="PyroDevil's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| PyroDevil reported|
|-|-|
|Trac ID|trac#7566|
|Type|release blocker: regression|
|Created|2014-08-23 18:13:39Z|

I have twisted (14.0.0) installed for python 3 on Archlinux.

Importing most **twisted.web.client** stuff fails because the **twisted.python.systemd** module doesn't exist.

```
>>> from twisted.web.client import HTTPClientFactory
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.4/site-packages/twisted/web/client.py", line 42, in <module>
    from twisted.internet.endpoints import TCP4ClientEndpoint, SSL4ClientEndpoint
  File "/usr/lib/python3.4/site-packages/twisted/internet/endpoints.py", line 34, in <module>
    from twisted.python.systemd import ListenFDs
ImportError: No module named 'twisted.python.systemd'
```

from [#6077](https://github.com/twisted/twisted/issues/6077) I get that this should be ported already to python 3, so its a regression. (I haven't tested it before)

<details><summary>Searchable metadata</summary>

```
trac-id__7566 7566
type__release_blocker__regression release blocker: regression
reporter__PyroDevil PyroDevil
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__web web
keywords__ 
time__1408817619965882 1408817619965882
changetime__1437418034448617 1437418034448617
version__None None
owner__None None
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl set status to `closed`|
|-|-|

This seems to be fixed now.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set status to `reopened`|
|-|-|

Fixed in [#7419](https://github.com/twisted/twisted/issues/7419), of which this appears to be a duplicate?

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set status to `closed`|
|-|-|
